### DISCUSSION TITLE
Is their a neu template for vue?

### DISCUSSION THREAD

**Discussion by JenuelDev** [neutralinojs/neutralinojs#400] [Q&A] [answered]
Is there a neu template for vue?

**Comment by shalithasuranga** [comment, marked as answer]
Hey.. in v2, we have removed all the templates and added https://github.com/neutralinojs/neutralinojs-minimal
The reason is that developers are following different patterns for frontend development. Now we can use any frontend framework. I think if you would like to use Vue, you can simply create a new Vue project in your application folder and update the resource path.

**Reply by JenuelDev** [reply]
Thanks for the quick reply, I will definitely check the link you shared, thanks thanks 👍

**Reply by JenuelDev** [reply]
Do you have a recommended video for this one?

**Reply by crstnmac** [reply]
@BroJenuel Did you get neutralinojs working with vue?

**Reply by JenuelDev** [reply]
@crstnmac not yet,, thanks for reminind,, I will try.. For now I am actually using electron since it has a lot of things. But I will be transferring to neutralino if neutralino if vue works fine...
